### PR TITLE
Add api document for authors endpoint

### PR DESCRIPTION
**Pull request by babygoat**

### CODE REVIEW

File: doc/news/authors.apib [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+### Get posts of an author [GET]
+
++ Response 200 (application/json)
+
+    + Attributes
+        + status: success (required)
+        + data
+            + meta (meta, fixed-type, required)
+            + records (array[MetaOfPost], fixed-type, required)
+
```
Comment (taylrj) [inline doc/news/authors.apib:86]: Just came up with one question:

When do we need status 204 or 200 with data containing empty records according to our design decisions(or principles)?

Comment (babygoat) [reply doc/news/authors.apib:86]: Both are OK for me. For now, it came from the request of Frontend. See [this thread](https://github.com/twreporter/go-api/pull/387#discussion_r443112412) for reference. We can have discussion and set the guideline here.

Comment (taylrj) [reply doc/news/authors.apib:86]: Thanks for the reference, current solution looks good to me.

**Comment by taylrj** [comment]
LGTM

File: doc/news/authors.apib [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Group Authors
+
+## Author List [/v2/authors{?keywords,sort,offset,limit}]
+A list contains information of the selected authors
+
+### Get a list of authors [GET]
+
++ Parameters
+    + keywords: `小明` (optional) - Keywords of the names of authors for searching
+    + sort: `-updated_at` (optional) - Field to soft by
```
[the remark this answers is not in the document]
Comment (babygoat) [reply doc/news/authors.apib]: updated.

**Pull request merged**